.github/workflows/documentation.yaml

```diff
@@ -76,6 +78,7 @@ jobs:
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
         with:
           persist-credentials: false
+          # Needed to detect missing redirects
```

Comment:
Leftover comment? Seems to apply to no particular line.

Reply:
hmm I must not have copied the `fetch-depth` line, copy pasta error, though it seems it's not required since `check` doesn't run the sphinx build itself.